src/frontend/js/modules/core.spec.js

```diff
@@ -420,4 +422,64 @@ describe('The Angular core module', function() {
       });
     });
   });
+
+  describe('The esnDomPurify filter', function() {
+    let esnDomPurify, $sce;
+
+    beforeEach(function() {
+      angular.mock.inject(function($window, $filter, _$sce_) {
+        $window.DOMPurify = DOMPurify;
+        esnDomPurify = $filter('esnDomPurify');
+        $sce = _$sce_;
+      });
+    });
+
+    it('should prefix ids and class, add div container and modify css selectors', function() {
+      const input =
+      `
+      <style>
+      .test-class {
+        color: #ffffff;
+      }
+      #test-id {
```

Comment:
can you also add more complex classes in your test like `div#test-id`, `#test-id div.class` something like that

Reply:
Added successfully :)